docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+# Scale and limits
+
+This document describes current scale and limits known in OSM as of 02/04/2021.
+
+## Considerations
+The scale limits documented here have to be put in light of current architecture. 
+Current architecture relies on a global broadcast mechanism that does not account for proxy configuration deltas, therefore all proxy configurations are computed and pushed upon any change.
+
+## Testing and measures
+We currently hold a single test which attempts to scale infinitely a topology subset, test proper traffic configuration between the new pods/services being deployed in the iteration, and stop if any failure is seen.
+It is also acknowledged that some of the scale constraints need to be addressed before it even makes sense to proceed with any additional scale testing, hence the lack of additional test scenarios.
+
+## Test
+The test was run in different OSM form factors, factoring in different amounts of RAM/CPU, to better qualify potential limits in case any of those were to be a constrain upon deployment.
+
+#### For all tests:
+- Change ID 9b829667f797879a0a4cd911eb08c4808e0f8083 (Thu Feb 4 13:38:37 2021)
+- 8 Node (1.18, nodes: 4vcpu 16Gb)
+- Envoy proxy log level Error
+- OSM controller log level Trace (reasons)
```

Comment:
Do we need `Trace` to define the baseline? Is `Trace` always going to be the default?

Reply:
Will reach offline....